### PR TITLE
report(flow): localized sample

### PR DESCRIPTION
**Pull request by adamraine**
In preparation for adding i18n support for the flow report, I'm adding a localized flow report to the sample reports along with some workflow changes.

#11313

### CODE REVIEW

File: build/build-sample-reports.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ const ReportGenerator = require('../report/generator/report-generator.js');
...
-  fs.readFileSync(
-    `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json`,
-    'utf-8'
-  )
+const flowResult = readJson(
+  `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json`
+);
+/** @type {LH.FlowResult} */
+const localizedFlowResult = readJson(
+  `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.en-XL.json`
```
Comment (patrickhulce) [inline build/build-sample-reports.js:26]: does this need to be separately checked in, could we use `swapLocale` just like the regular LHRs?

File: lighthouse-core/scripts/update-flow-fixtures.js [hunk 2]
```diff
@@ -7,42 +7,74 @@
...
+  fs.writeFileSync(
+    `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json`,
+    JSON.stringify(flowResult, null, 2)
+  );
+}
+
+/**
+ * @param {import('puppeteer').Browser} browser
+ */
+async function buildLocalizedFlowResult(browser) {
```
Comment (patrickhulce) [inline lighthouse-core/scripts/update-flow-fixtures.js:47]: same here, let's use swapLocale instead? and/or create a flow-specific one

**Review by patrickhulce — CHANGES_REQUESTED** [review]
glad we're getting on this! :)

File: lighthouse-core/scripts/update-flow-fixtures.js (hunk 2 above)
Comment (adamraine) [reply lighthouse-core/scripts/update-flow-fixtures.js:47]: I think I'll just revert this one

File: lighthouse-core/lib/i18n/swap-flow-locale.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+const swapLocale = require('./swap-locale.js');
+
+/**
+ * @param {LH.FlowResult} flowResult
+ * @param {LH.Locale} locale
+ */
+module.exports = function swapFlowLocale(flowResult, locale) {
+  const localizedFlowResult = JSON.parse(JSON.stringify(flowResult));
+  localizedFlowResult.steps = flowResult.steps.map(s => {
+    return {...s, lhr: swapLocale(s.lhr, locale).lhr};
```
Comment (patrickhulce) [inline lighthouse-core/lib/i18n/swap-flow-locale.js:17]: c'mon man, we're not even close to hitting line limits ;)

```suggestion
  localizedFlowResult.steps = flowResult.steps.map(step => {
    return {...step, lhr: swapLocale(step.lhr, locale).lhr};
```

**Review by patrickhulce — APPROVED** [review]

**Pull request merged**